tests/integration_tests/tests/agentless_tests/deployment_update/test_deployment_update_modification.py

```diff
@@ -388,3 +390,45 @@ def test_modify_description(self):
 
         deployment = self.client.deployments.get(dep_update.deployment_id)
         self.assertRegexpMatches(deployment['description'], 'new description')
+
+    def test_modify_inputs_ops_order(self):
+        """Verify that update workflow executes uninstall and install."""
+        deployment, _ = \
+            self._deploy_and_get_modified_bp_path('modify_inputs')
+        self.assertEquals(deployment['inputs'], {
+                          u'test_list': u'initial_input'})
+
+        dep_update = \
+            self.client.deployment_updates.update_with_existing_blueprint(
+                deployment.id,
+                inputs={u'test_list': [u'update_input1', u'update_input2']},
+            )
+        # assert that 'update' workflow was executed
+        self._wait_for_execution_to_terminate(deployment.id, 'update')
+        self._wait_for_successful_state(dep_update.id)
+        execution_ids = [en.id for en in self.client.executions.list(
+            deployment_id=deployment.id,
+            workflow_id='update',
+            status='terminated',
+        )]
+        self.assertEquals(len(execution_ids), 1)
+
+        # verify if inputs have been updated
+        deployment = self.client.deployments.get(dep_update.deployment_id)
+        self.assertEquals(deployment['inputs'], {
```

Comment:
plz `assertEqual`, sans `s` - the old "equals" alias logs a warning at runtime :(